app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
As per this requirement, Please add `:confirmable` module here to make email confirmation.
[Hint]: See this [video](https://www.youtube.com/watch?v=nun18zM8Gq0) for more info.